doc/sphinx/how-to/how-to-enable-remote-desktop.md

```diff
@@ -0,0 +1,29 @@
+# How to Enable Remote Desktop
+Mir supports remote desktops via the VNC protocol. To demo this, you'll use
+`wayvnc` - a VNC server. 
```

Comment:
Not blocking for this, as it is pre-existing, but may be worth fixing before establishing a path:

This doesn't do all of "Enable Remote Desktop" - the _remote_ part is _not_ addressed. What it does do is demonstrate using a VNC server with Mir.